### ISSUE TITLE
[1.14.4_HD_U_F2] Bug constantly loads chunks into ServerChunkCache until ram is full, interacting with portals/chests stops working

### ISSUE THREAD

**Issue by Jadasco** [sp614x/optifine#2712]
Hey, I'm pretty confident this is an optifine bug because it hasn't happened to me in extensive play on 1.14.4 vanilla but it has 3 times on optifine's latest release.

the game runs smoothly for a while during this bug, but ServerChunkCache in F3 will constantly increase until the allocated RAM fills up and performance drops. I've seen it get to 27,000 before. Picking up items off the ground doesn't seem to put them in my inventory anymore, chests won't open and beds can't be used, lots of stuff breaks like that. 

Even worse, it's no longer possible to save - the game gets stuck on "saving world" forever. It's also not possible to close the game without ending the java task.

**Comment by sp614x** [comment]
Running with Fabric or standalone?
Single-player or multiplayer?
Can you post an F3 screenshot?

**Comment by Jadasco** [comment]
S

> 
> 
> Running with Fabric or standalone?
> Single-player or multiplayer?
> Can you post an F3 screenshot?

Standalone, Single player and unfortunately I don't have one of the bug in action. I'll attach one as soon as I get one.

**Comment by Jadasco** [comment]
one thing I've noticed looking at the number is that the loading is continuous, one by one and around 40/second, and not in discrete 10+ numbers like it would be if the player was moving round normally

**Comment by Jadasco** [comment]
> 
> 
> Running with Fabric or standalone?
> Single-player or multiplayer?
> Can you post an F3 screenshot?
here you go, bug in action. The mobs aren't moving and I can't damage them
![image](https://user-images.githubusercontent.com/25494419/62290828-21490280-b45a-11e9-982e-5fe01c554027.png)

**Comment by Jadasco** [comment]
probably important update? I encountered this today without optifine, after about 10 hours in game cumulatively since I swapped version back to stock. I was able to save/return to the main menu without ending the process, unlike when it happened with optifine. I have no idea what that means or why it'd be more common in optifine, but it's possible that it's mojang's fault and nothing to do with it. Sorry for the confusion
